gatorgrade/input/in_file_path.py

```diff
@@ -1,6 +1,6 @@
 """Generates a list of commands to be run through gatorgrader."""
 import yaml
-import set_up_shell
+from set_up_shell import *
```

Comment:
Avoid using `*` imports -- instead just list directly what you want to import, to make the usage clear.

Reply:
Okay thank you!